src/main/java/no/nav/veilarbaktivitet/aktivitetskort/dto/aktivitetskort/Sentiment.java

```diff
@@ -0,0 +1,12 @@
+package no.nav.veilarbaktivitet.aktivitetskort.dto.aktivitetskort;
+
+/**
+ * Stylinghint for etiketter
+ */
+public enum Sentiment {
+    POSITIVE, // Fått tilbud, Klar for oppstart, Påbegynt, Påmeldt? (gode nyheter for brukeren)
+    NEGATIVE, // Ikke fått jobben, Avbrutt, Annulert, Fått avslag (dårlige nyheter for brukeren)
+    ACTION, // Venter svar fra deg (Venter handling fra brukeren)
```

Comment:
Ble vi ikke enige om å ikke kalle det ACTION siden de blir for vanskelig å skille den fra WAITING